backend/src/main/java/gov/cdc/usds/simplereport/config/SecurityConfiguration.java

```diff
@@ -37,41 +37,36 @@
 @ConditionalOnWebApplication
 public class SecurityConfiguration extends WebSecurityConfigurerAdapter {
 
-    public static final String SAVED_REQUEST_HEADER = "SPRING_SECURITY_SAVED_REQUEST";
-    private static final Logger LOG = LoggerFactory.getLogger(SecurityConfiguration.class);
+	public static final String SAVED_REQUEST_HEADER = "SPRING_SECURITY_SAVED_REQUEST";
+	private static final Logger LOG = LoggerFactory.getLogger(SecurityConfiguration.class);
 
-    public interface OktaAttributes {
-        public static String EMAIL = "email";
+	public interface OktaAttributes {
+		public static String EMAIL = "email";
 		public static String FIRST_NAME = "given_name";
 		public static String LAST_NAME = "family_name";
-    }
+	}
 
-    @Override
-    protected void configure(HttpSecurity http) throws Exception {
-        http
-                .authorizeRequests()
-                .antMatchers("/").permitAll()
-                .antMatchers(HttpMethod.GET, WebConfiguration.HEALTH_CHECK).permitAll()
-                .antMatchers(HttpMethod.OPTIONS, "/**").permitAll()
+	@Override
+	protected void configure(HttpSecurity http) throws Exception {
+		http.authorizeRequests().antMatchers("/").permitAll().antMatchers(HttpMethod.GET, WebConfiguration.HEALTH_CHECK)
```

Comment:
Please don't reformat code like this.

Reply:
Sorry, I didn't mean to! I've got VSCode set to auto-format on save. Happy to revert these